contracts/smart-contract-wallet/paymasters/verifying/singleton/VerifyingSingletonPaymaster.sol

```diff
@@ -248,9 +252,10 @@ contract VerifyingSingletonPaymaster is
     ) internal virtual override {
         PaymasterContext memory data = context._decodePaymasterContext();
         address extractedPaymasterId = data.paymasterId;
+        uint256 effectiveGasPrice = block.basefee + data.maxPriorityFeePerGas;
```

Comment:
We should use 
uint256 effectiveGasPrice = min(data.maxFeePerGas, data.maxPriorityFeePerGas + block.basefee)

function min(uint256 a, uint256 b) internal pure returns (uint256) {
        return a < b ? a : b;
    }

Reply:
done